File: backend/components/metiers/administration_publique/utils/rbac/samples/sample_usage.py
# sample_usage.py
"""Exemple d'utilisation du module RBAC (Python)"""
from ..core import rbac_core
import json
with open('sample_rbac_data.json') as f:
    data = json.load(f)

print('Peut lire ?', rbac_core.check_permission(data['user'], 'read'))
print('Rôles:', rbac_core.get_user_roles(data['user']))

File: backend/components/metiers/administration_publique/utils/rbac/samples/sample_usage.js
// sample_usage.js
// Exemple d'utilisation du module RBAC (JS)
const rbac = require('../core/rbac_core');
const data = require('./sample_rbac_data.json');

// Vérification de permission
console.log('Peut lire ?', rbac.checkPermission(data.user, 'read'));

// Rôles de l'utilisateur
console.log('Rôles:', rbac.getUserRoles(data.user));
